scripts/mutate.py

```diff
@@ -162,6 +163,11 @@ def mutate(
         # seem to happen in the checker codebase though...
         if stripped.startswith("(*") or stripped.startswith("*"):
             continue
+        # Super basic logic for ignoring assertions. This might be overly
+        # aggressive (e.g. in cases with an inline comment talking about assertions), but
+        # this doesn't seem to be a common case.
+        if "assert" in stripped:
+            continue
```

Comment:
I think this is probably OK as-is for now, but if you really want to match the way assertions are treated at the moment by the rest of the pipeline you can see if `stripped` matches the regexps we have in [`generate-ligo.sh`](https://github.com/tezos-checker/checker/blob/ad697ff76c97489d218265b91810df88c3ab74d7/scripts/generate-ligo.sh#L77-L82):
```
    ...
    # delete assertions
    sed 's/^ *assert .*//g' |
    sed 's/^ *assert_burrow_invariants .*//g' |
    sed 's/^ *assert_checker_invariants .*//g' |
    sed 's/^ *assert_liquidation_auction_invariants .*//g' |
    sed 's/^ *assert_burrow_slices_invariants .*//g' |
    ...
```

Reply:
Good point, that makes sense. Since the cruder approach is actually a bit faster than using a regex I might leave it as-is for now since I don't think it is excluding too many (if any) legitimate lines which should be tested.